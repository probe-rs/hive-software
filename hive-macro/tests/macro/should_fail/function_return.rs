#[hive_macro::hive_test]
fn test_function() -> u8 {}

fn main() {}
