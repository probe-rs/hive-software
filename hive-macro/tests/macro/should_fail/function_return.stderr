error: Non () return type found

         = help: Hive testfunctions should not have any return value.
         = note: example: fn my_fancy_test(test_channel: &mut dyn TestChannelHandle, session: &mut Session);

 --> tests/macro/should_fail/function_return.rs:2:20
  |
2 | fn test_function() -> u8 {}
  |                    ^^
